pkg/karmadactl/promote.go

```diff
@@ -85,8 +86,8 @@ func NewCmdPromote(f util.Factory, parentCommand string) *cobra.Command {
 	flag := cmd.Flags()
 	opts.AddFlags(flag)
 
-	flag.StringVar(defaultConfigFlags.KubeConfig, "kubeconfig", *defaultConfigFlags.KubeConfig, "Path to the kubeconfig file to use for CLI requests.")
-	flag.StringVar(defaultConfigFlags.Context, "karmada-context", *defaultConfigFlags.Context, "The name of the kubeconfig context to use")
+	flag.StringVar(options.DefaultConfigFlags.KubeConfig, "kubeconfig", *options.DefaultConfigFlags.KubeConfig, "Path to the kubeconfig file to use for CLI requests.")
```

Comment:
Why not `options.AddKubeConfigFlags()`?

Reply:
Sorry for the omission, now it has been replaced.